README.md

```diff
@@ -19,11 +19,15 @@ Experts team using HR Support:
 
 Please read the license terms applicable to this template [here](https://github.com/OfficeDev/microsoft-teams-hrsupport-app/blob/master/LICENSE). In addition to these terms, you agree to the following:
 
-- You are responsible for complying with all applicable privacy and security regulations, as well as all internal privacy and security policies of your company. You must also include your own privacy statement and terms of use for your app if you choose to deploy or share it broadly
-
-- Use and handling of any personal data collected by your app is your responsibility. Microsoft will not have any access to data collected through your app, and therefore is not responsible for any data related incidents
-
-- Any Microsoft trademarks and logos included in this repository are property of Microsoft and should not be reused, redistributed, modified, repurposed, or otherwise altered or used outside of this repository
+ - You are responsible for complying with privacy and security regulations applicable to your app.

```

Comment:
Typo: “are responsible”

Reply:
I don't seem to get this. Can you indicate where the typo is?

Reply:
I did. “You are responsible”, not “You areresponsible”.